.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1 @@
+blank_issues_enabled: false
```

Comment:
I feel like blank issues might have some place, though this is totally up for debate. If something doesn't need a description, then there's no reason to force people to write one. An example of this might be in our Project page, where someone could add `Create a Header`. Adding a description doesn't add much value, because the exact layout for the header already exists on Figma. 

**If this config doesn't stop projects integration like how I described, feel free to ignore this.**

Reply:
The main reason I wanted to present these was to help with organizing all the new issues. I thought we might want to guide people into what is needed instead of leaving them up to their own devices. I want to avoid being in triage hell where there are all these incomplete tickets lying around that no one is ever going to deal with. Maybe I'm over complicating this. Let's talk about it on Tuesday.

Reply:
for now though I will set that to true and we can revisit later if it becomes an issue

Reply:
> The main reason I wanted to present these was to help with organizing all the new issues. I thought we might want to guide people into what is needed instead of leaving them up to their own devices. I want to avoid being in triage hell where there are all these incomplete tickets lying around that no one is ever going to deal with. Maybe I'm over complicating this. Let's talk about it on Tuesday.

I think we can prefer descriptive issues, but allow blank ones for more general things. If I'm asking someone to build an event component, for example, I'll add an actual description explaining what I mean.